packages/cli/src/index.js

```diff
@@ -181,6 +183,29 @@ async function runYargs() {
     .option('cwd', {
       describe: 'Working directory to use (where `redwood.toml` is located)',
     })
+    .option('include-env', {
+      describe: 'Load additional .env files. These are incremental',
+      array: true,
+    })
+    .example(
+      'yarn rw exec MigrateUsers --include-env prod --include-env stripe-prod',
+      '"Run a script, and also include .env.prod and .env.stripe-prod"'
+    )
+    .middleware(
+      [
+        // We've already handled `cwd` above, but it may still be in `argv`.
+        // We don't need it anymore so let's get rid of it.
+        // Likewise for `telemetry`.
+        (argv) => {
+          delete argv.cwd
+          delete argv.telemetry
+        },
+        telemetry && telemetryMiddleware,
+        updateCheck.isEnabled() && updateCheck.updateCheckMiddleware,
```

Comment:
These two (telemetry and updateCheck) seem to be duplicated from above; will push a commit deduping the middleware here

Reply:
Done in https://github.com/redwoodjs/redwood/pull/9961/commits/6b679d03b105c371a391c6f5a147bb3577af7a17